src/Xamarin.Android.Tools.AndroidSdk/Properties/AssemblyInfo.cs

```diff
@@ -12,4 +12,5 @@
 
 [assembly: AssemblyVersion ("1.0.0.0")]
 [assembly: AssemblyFileVersion ("1.0.0.0")]
+[assembly: AssemblyInformationalVersion ("1.0.99.0")]
```

Comment:
Should this version differ from `AssemblyVersion`, above?

Relatedly -- which increases complexity immensely, so probably *shouldn't* be done but *could* be considered -- do we want the assembly version values to match the nuget version values?  The only way to do this would be to make `AssemblyInfo.cs` a generated file, so that it can contain the appropriate `$commitsSinceVersionChange` value.